include/grpcpp/security/tls_credentials_options.h

```diff
@@ -97,6 +97,11 @@ class TlsCredentialsOptions {
   // verifiers other than the host name verifier is used.
   void set_check_call_host(bool check_call_host);
 
+  // If set, gRPC will read all hashed x.509 CRL files in the directory and
+  // enforce the CRL files on all TLS handshakes. Only supported for OpenSSL
+  // version > 1.1.
+  void set_crl_directory(const std::string& path);
```

Comment:
Please add a comment indicating that this API is likely to change in the future, and a TODO reminding us to address this before de-experimentalizing the TlsCreds API.